connector/connect/README.md

```diff
@@ -21,6 +21,17 @@ user experience across all languages. Please follow the below guidelines:
 
 Python-specific development guidelines are located in [python/docs/source/development/testing.rst](https://github.com/apache/spark/blob/master/python/docs/source/development/testing.rst) that is published at [Development tab](https://spark.apache.org/docs/latest/api/python/development/index.html) in PySpark documentation.
 
```

Comment:
Would it be useful to add a README in this folder and link to this README?

Reply:
I think that folder is `rst` based and published to the web documentation and not markdown based, so I think README.md doesn't fit there.
But I was surprised that the documentation there doesn't mention getting dependencies from `dev/requirements.txt`. But I'm not really familiar with python dev and these docs, maybe someone more familiar should have a better idea what would be the best place for it? @HyukjinKwon ?